stable/acs-engine-autoscaler/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+type: Opaque
```

Comment:
Minor nit: for consistency's sake you should add the labels to all K8s resources managed by Helm (again, using `labels.standard`).